app/grandchallenge/archives/views.py

```diff
@@ -558,6 +564,115 @@ def get_serializer_class(self):
             return ArchiveItemSerializer
 
 
+class ArchiveItemJobListView(
+    LoginRequiredMixin,
+    ObjectPermissionRequiredMixin,
+    JobsList,
+):
+    model = Job
+    template_name = "archives/archive_item_job_list.html"
+    row_template = "algorithms/job_list_row.html"
+
+    search_fields = [
+        *JobsList.search_fields,
+        # Outputs
+        "outputs__interface__title",
+        "outputs__interface__description",
+        # Algorithm
+        "algorithm_image__algorithm__title",
+        "algorithm_image__algorithm__description",
+        "algorithm_image__comment",
+    ]
+
+    permission_required = (
+        f"{Archive._meta.app_label}.view_{ArchiveItem._meta.model_name}"
+    )
+    raise_exception = True
+
+    login_url = reverse_lazy("account_login")
+
+    @cached_property
+    def archive_item(self):
+        return get_object_or_404(ArchiveItem, pk=self.kwargs["pk"])
+
+    def get_permission_object(self):
+        return self.archive_item
+
+    def get_queryset(self):
```

Comment:
This should include a call to `super()`. Currently that will fail because you inherited from `JobList` which expects an algorithm slug, not an archive slug, but as I argue above, I would inherit directly from `PaginatedListVIew` instead.

Reply:
Somehow, whenever I select a row to comment on, it also selects a few rows before, making the comment less clear. Sorry about that! This refers to the `get_queryset` method, not `get_permission_object`